react-components/src/hooks/types.ts

```diff
@@ -61,6 +62,7 @@ export type SceneModelsProperties = Transformation3d & {
 
 export type SceneConfigurationProperties = {
   name: string;
+  skybox?: DmsUniqueIdentifier;
```

Comment:
Should there be a difference between `SceneConfigurationProperties` and `SceneData`? Could they be combined?

Reply:
Also - should groundPlane be added to this configuration too?

Reply:
I think that they should be kept separate, as `SceneConfigurationProperties` is the properties of the FDM instnaces, whereas the `SceneData` is all data associated with a Scene, including connected data.

Reply:
Groundplanes is already in the `SceneData`